frontend/packages/topology/src/components/page/TopologyPageToolbar.tsx

```diff
@@ -37,6 +40,7 @@ const TopologyPageToolbar: React.FC<TopologyPageToolbarProps> = observer(
             <Button
               type="button"
               variant="link"
+              aria-label={t('topology~View shortcuts')}
```

Comment:
Fairly certain that `aria-label` for a button containing text is unnecessary unless the `aria-label` has more descriptive text which differs from the element text.

Reply:
Here we may need the UX input for text.

Reply:
@vikram-raj we can change it to `View topology shortcuts` or something similar to make it more descriptive.
cc: @beaumorley

Reply:
In most cases the button text is sufficient and since we already decided that this text 'view shortcuts' is sufficient for the normal user.

Reply:
I still do not feel it necessary to add an aria-label here unless you can say that UX says otherwise.

Reply:
We have an accessibility issue with this shortcut button that is the reason I am adding `aria-label` to this.

Reply:
@vikram-raj are you sure we have an issue? The two linked jira issues do not mention the shortcuts button.

Please share the issue.

Reply:
It is the issue https://issues.redhat.com/browse/ODC-4949

Reply:
Fairly certain they actually meant the `!` icon button because it has no aria-label or inner text.
There are no accessibility violations according to a tool i use (axe) to test the `? view shortcuts` button where as the `!` icon button is missing text.

Did you test with an accessibility tool to see the violation before and after?

Reply:
Removed this `aria-label` .